backend/tests/TestUtils/TestUtils.fs

```diff
@@ -566,7 +566,7 @@ module Expect =
     // expressions with no values
     | EUnit _, EUnit _ -> ()
     // expressions with single string values
-    | EString (_, v), EString (_, v') -> ()
+    | EString (_, s), EString (_, s') -> check path s s'
```

Comment:
So what's going on here is that `s` is a list of string parts and interpolated expressions. We need to compare each of them, using `check` to check equality for the string parts, and `eq` to check equality on the interpolated expressions.